src/styles/blocks/watch.scss

```diff
@@ -1,50 +1,40 @@
 .watch {
   position: absolute;
-  @include circle ($watch-size);
-  @include shift ($watch-centering);
+  @include circle($watch-size);
+  @include shift($watch-centering);
   display: flex;
   justify-content: center;
   align-items: flex-end;
   background-color: $blue;
-  border-radius: 100%;
+  border-radius: 50%;
   border: $border-width solid $yellow;
   transform: translate(-50%, -50%);
 
-  &__hand {
+  &__arrow {
     position: absolute;
     bottom: $watch-centering;
 
     &--second {
-      @include second-hand ($second-hand-height);
-      background-color: #ffd700;
+      @include second-arrow($second-arrow-height);
```

Comment:
also delete parameter `$second-arrow-height` from here, as your mixin doesn't use it